run_snafu.py

```diff
@@ -46,7 +46,7 @@ def main():
         '-t', '--tool', help='Provide tool name')
     index_args, unknown = parser.parse_known_args()
     index_args.index_results = False
-    index_args.prefix = "snafu-%s" % index_args.tool
+    index_args.prefix = "ripsaw-%s" % index_args.tool
```

Comment:
The default settings should be snafu not ripsaw, this was deliberately asked for by @aakarshg.

Reply:
I did not know that, will fix.

Reply:
done